### ISSUE TITLE
rsp.headers is not a function ERROR

### ISSUE THREAD

**Issue by viniciusfdev** [ryantxu/ajax-panel#66]
My grafana version is 6.7.0
My ajax plugin is the latest.
I am trying to use the plugin and i am getting the error in dev tools console after call a post request, the request is sent normally and the response is get normally to.

![image](https://user-images.githubusercontent.com/28795085/83896185-1fab3380-a72a-11ea-9a86-09e3ee8f9a78.png)

**Comment by amotl** [comment]
Dear Vinicius,

this is probably a duplicate of #60 and might have been fixed through #64 already. If you will be able to build this plugin for yourself, it might help you along.

With kind regards,
Andreas.

**Comment by viniciusfdev** [comment]
@amotl Thank you so much, so, until they merge the commit with master i did the same modifications in the file that the commit do, but i am having problem building the plugin:
This is my error, any orientation will be very helpful.
![image](https://user-images.githubusercontent.com/28795085/83916588-d4098180-a74b-11ea-9074-2ec8c6c3de6c.png)

**Comment by amotl** [comment]
Dear Vinicius,

I just tried and had no problems building the vanilla master using `yarn install` followed by `yarn build`.

The Prettier is sometimes a bit picky, so it might just be this guy complaining.

You might well want to directly clone https://github.com/BlackLight/ajax-panel. Thanks again for the fix, @BlackLight.

With kind regards,
Andreas.

**Comment by amotl** [comment]
Indeed, after switching to @BlackLight's fork, I am also getting (different) errors:
```
  ERROR in /Users/amo/dev/panodata/sources/ajax-panel/src/module.ts
  [tsl] ERROR in /Users/amo/dev/panodata/sources/ajax-panel/src/module.ts(499,31)
        TS2554: Expected 1 arguments, but got 0.

  ERROR in /Users/amo/dev/panodata/sources/ajax-panel/src/module.ts
  [tsl] ERROR in /Users/amo/dev/panodata/sources/ajax-panel/src/module.ts(507,18)
        TS2349: This expression is not callable.
    Type 'typeof import("/Users/amo/dev/panodata/sources/ajax-panel/node_modules/@grafana/data/index")' has no call signatures.
```

**Comment by viniciusfdev** [comment]
Dear @amotl ,
I tried to use BlackLigh solution, but for me this modifications only remove the 'type function error'. So i made some modifications based on this https://github.com/ryantxu/ajax-panel/issues/60#issuecomment-626465545
This modifications was a solution to Panel refresh problem, now when he receive the response from server he is refreshed, 
it wasn't happening before. I will post my solution and make a pull request as soon as i can.

**Comment by viniciusfdev** [comment]
I made a solution for this issue and two other issues i think!
https://github.com/ryantxu/ajax-panel/issues/60#issue-591703784
https://github.com/ryantxu/ajax-panel/issues/65#issue-628320015
my repo:
https://github.com/viniciusfdev/ajax-panel.git

@robertpascale i think this will help you!

**Comment by robertpascale** [comment]
Hi @viniciusfdev ,
That's great - but I'm not sure how I can use your patch. I currently run grafana in a docker container on a NAS box, and I just add the plugin via the docker environment variable as follows:
GF_INSTALL_PLUGINS=ryantxu-ajax-panel
I don't have any way of compiling my own and even if I did, I'm not sure how I would embed it in the docker container.
Hopefully @ryantxu  may action the pull request and roll it in to the next production release.

**Comment by viniciusfdev** [comment]
Dear @robertpascale,
I'm also running grafana on the docker, if you really need this fix as early as I do, just clone my repository, install all the dependencies needed to create the grafana plugin like @ grafana / tools and yarn, run 'yarn install' and 'yarn build ', doing this, a folder called' dist 'will appear, the' dist 'folder is used by grafana to create a plugin, so you need to put this' dist' folder inside another folder as' ajax-plugin 'and put inside the directory / data / plugis in the grafana container (with docker cp or using volumes).

**Comment by kavdev** [comment]
running into this with grafana 7.1.1
